dataflow/src/main/java/org/checkerframework/dataflow/analysis/FlowExpressions.java

```diff
@@ -447,8 +447,22 @@ private static Receiver internalReprOfMemberSelect(
     }
 
     /**
-     * The poorly-named Receiver class is actually a Java AST. Each subclass represents a different
-     * type of expression, such as MethodCall, ArrayAccess, LocalVariable, etc.
+     * This class represents a Java expression and its type. However, it does not represent all
+     * possible Java expressions (for example, it does not represent a ternary expression; use
+     * {@link FlowExpressions.Unknown} for unrepresentable expressions). Furthermore, it supports
```

Comment:
Our "Java" expression language supports "#1", but it is parsed into a `LocalVariable`.  So this comment here doesn't really make sense.

Reply:
@smillst Can you clarify your concern?  Do you mean that it doesn't make sense to call `#1` a formal parameter?  From the perspective of a user that's what it is used for.  The Java compiler and JVM do treat formal parameters as local variables, but I think of that as an implementation detail.  Can you suggest alternate wording if this comment is misleading?

Reply:
The Javadoc seems to imply that there are Receiver classes for the extensions to the Java language, but there aren't.  `#1` is represented by a local variable and `<self>` is  converted before parsing by the Lock Checker.

Reply:
Thanks for clarifying.